Src/Support/Google.Apis.Auth.AspNetCore3.IntegrationTests/Controllers/HomeController.cs

```diff
@@ -80,8 +80,17 @@ public async Task<IActionResult> ScopeListing([FromServices] IGoogleAuthProvider
             return View(currentScopes);
         }
 
-        // Test showing use of incremental auth.
-        // This attribute states that the listed scope(s) must be authorized in the handler.
+        /// <summary>
+        /// List the Storage buckets the authenticated user has access to on this app's GCP project.
+        /// Specifying the <see cref="GoogleScopedAuthorizeAttribute"> will guarantee that the code
+        /// executes only if the user is authenticated and has granted the
+        /// <see cref="StorageService.ScopeConstants.CloudPlatformReadOnly"/> to this application.
```

Comment:
```suggestion
        /// executes only if the user is authenticated and has granted the scope specified
        /// in the attribute to this application.
```

(Otherwise it sounds like it only works with that scope.)

Reply:
Ah yes, done.